src/main/io/serial_msp.c

```diff
@@ -1868,6 +1875,12 @@ static bool processInCommand(void)
         masterConfig.baro_hardware = read8();
         masterConfig.mag_hardware = read8();
         break;
+        
+    case MSP_SET_NAME:
+        for (i = 0; i < MAX_NAME_LENGTH; i++) {
```

Comment:
Should you add the length here, and only set the amount needed (set to null everything after it)?